src/pudl/metadata/fields.py

```diff
@@ -2680,9 +2680,9 @@
         ),
         "unit": "MW",
     },
-    "net_demand_mwh": {
+    "net_demand_forecast_mwh": {
         "type": "number",
-        "description": "Net electricity demand for the specified period in megawatt-hours (MWh).",
+        "description": "Net forecasted electricity demand for the specific period in megawatt-hours (MWh).",
```

Comment:
I love the addition of forecast in here. it makes it much more clear.

not blocking... but is there a difference between this and `demand_forecast_mwh`?

Reply:
Yes, `demand_forecast_mwh` is from 930 and is a day-ahead forecast. This is a year-ahead forecast. The 930 value is also demand vs. the 714 value which is net demand.

Reply:
We could probably add a day-ahead suffix or something to the 930 value! What do you think?

Reply:
_Technically_ for the FERC714 table it's net_load vs net_demand, but they are sooo similar idk if it matters. But it's called net load in the FERC taxonomy.

Reply:
yea it makes sense to me to stay true to the source taxonomy unless we learn its wrong somehow, but that's not the case here